function-images/springboot/README.md

```diff
@@ -0,0 +1,3 @@
+# vHive-bench-java
+
+This project was bootstrapped using Spring Initializr at start.spring.io
```

Comment:
typo Initializr? the link must be formatted as a link

Reply:
Not a typo

Reply:
then please add this to configs/.wordlist* file so that the spellchecker is happy